docs/introduction/index.rst

```diff
@@ -10,13 +10,13 @@ Among others, it can be used in combination with `Open Zaak`_ which implements t
 
 .. _`Common Ground`: https://commonground.nl/
 .. _`Open Zaak`: https://github.com/open-zaak/open-zaak/
-.. _`VNG API standard for the "Notificaties API"`: https://zaakgerichtwerken.vng.cloud/
-.. _`VNG standards for "API's voor Zaakgericht werken"`: https://zaakgerichtwerken.vng.cloud/
+.. _`VNG API standard for the "Notificaties API"`: https://vng.nl/projecten/zaakgericht-werken
```

Comment:
```suggestion
.. _`VNG API standard for the "Notificaties API"`: https://vng-realisatie.github.io/gemma-zaken/
```
Description and URL don't seem to match, but I'm not sure what exactly is going on here with the different labels for the same URLs.